src/binding.cc

```diff
@@ -1967,17 +1968,33 @@ v8::StartupData v8__SnapshotCreator__CreateBlob(
   return self->CreateBlob(function_code_handling);
 }
 
-v8::Platform* v8__platform__NewDefaultPlatform() {
-  // TODO(bnoordhuis) Support optional arguments.
-  return v8::platform::NewDefaultPlatform().release();
+void v8__Platform__CONSTRUCT(
+  uninit_t<v8::platform::DefaultPlatform>* buf,
+  int thread_pool_size,
+  bool idle_task_support
+) {
+  static_assert(sizeof(v8::platform::DefaultPlatform) <= 20 * sizeof(size_t), "Platform size mismatch");
```

Comment:
Can you wrap at 80 columns in this file? Thanks.

Reply:
We don't check it in CI but this is formatted with clang-format 

https://github.com/denoland/rusty_v8/blob/main/.clang-format